source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -100,6 +100,7 @@ class HttpConnectionManagerConfig : Logger::Loggable<Logger::Id::config>,
   std::chrono::milliseconds drainTimeout() override { return drain_timeout_; }
   FilterChainFactory& filterFactory() override { return *this; }
   bool generateRequestId() override { return generate_request_id_; }
+  const bool edgeAcceptRequestId() override { return edge_accept_request_id_; }
```

Comment:
sorry I meant:
```
bool edgeAcceptRequestId() const override { return edge_accept_request_id_; }
```
etc. That `const` after the arg-list is a way of guaranteeing that the contents of the class instance will not be modified by the function.